fc-community-backend/apps/api/src/app.service.ts

```diff
@@ -18,7 +18,7 @@ export class AppService {
     const fc = new FactChainBackend(config);
     const ns = new NoteService(fc, fc);
     const notes = await ns.getNotes(
-      (_postUrl, _) => _postUrl === postUrl,
+      (_postUrl, _) => _postUrl === postUrl.replace("twitter", "x"),
```

Comment:
* Yep, you're right; I was aware of replacing nth time rather than 1. I didn't want to commit this, but if we stored a Twitter URL once, don't we need to replace all URLs when getting notes? 

* I don't think we need the regex since `replace` only affects the first occurrence matched.